sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -221,7 +230,8 @@ class AdaptiveQueryExecSuite
       val bhj = findTopLevelBroadcastHashJoin(adaptivePlan)
       assert(bhj.size == 2)
       val ex = findReusedExchange(adaptivePlan)
-      assert(ex.size == 1)
+      // The ReusedExchange is hidden in LocalShuffleReaderExec
```

Comment:
We should update `findReusedExchange` and make it go through `LocalShuffleReaderExec`